unattended-upgrade

```diff
@@ -1655,6 +1696,37 @@ def run(options,             # type: Options
     if should_stop():
         return UnattendedUpgradesResult(False)
 
+    # check to see if want to auto-upgrade the devel release
+    if apt_pkg.config.find("Unattended-Upgrade::DevRelease") == "auto":
+        try:
+            import distro_info
```

Comment:
Having a test for this would be awesome, maybe we can mock distro_info relatively easily?